app/main.py

```diff
@@ -1,3 +1,61 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
no need to write Distnace with quotes in type annotation, it will work fine without them

Reply:
It gives an error "Unresolved reference 'Distance' ", because we do it directly in the body of the class

Reply:
You could import annotations from __future__ to solve the issue with type not being recognized inside class definition